prac_06/programming_language.py

```diff
@@ -0,0 +1,16 @@
+class ProgrammingLanguage:
+    def __init__(self, name, typing, reflection=False, year=0):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        if self.reflection:
+            return True
+        else:
+            return False
```

Comment:
since you're actually checking whether reflection is true or false, you could just directly return self.reflection. However the function is meant to check whether the typing type is dynamic, so you'd be checking whether self.typing == "Dynamic". And since the '==' operator returns true or false, instead of using and if statement, you can directly return self.typing == "Dynamic"

Reply:
Oh whoops, I clearly didn't read the question closely haha, all fixed now